src/main/java/helper/Parser.java

```diff
@@ -0,0 +1,44 @@
+package helper;
+
+import helper.command.*;
+
+/**
+ * Parsing class
+ */
+public class Parser {
+    /**
+     * Parses the user input for duke
+     * @param s User input
+     * @return Command to execute
+     * @throws DukeException
+     */
+    public Command parse(String s) throws DukeException {
```

Comment:
I am not sure parameter s is compatible with our coding standard. But a meaningful naming will always make it better. What do you think?